.gitignore

```diff
@@ -0,0 +1,25 @@
+/Makefile
+/Makefile.in
+/aclocal.m4
+/autom4te.cache/
+/config/
+/config.log
+/config.status
+/configure
+/extensions/CoverflowAltTab@dmo60.de/Makefile
```

Comment:
This line can be removed after removing the leading slash from the Makefile entry above in the file.